jib-core/src/test/java/com/google/cloud/tools/jib/http/ConnectionTest.java

```diff
@@ -58,6 +58,8 @@
   private Request fakeRequest;
   private HttpResponse mockHttpResponse;
 
+  private long byteCount = 0;
```

Comment:
Use a local `LongAdder` like others and remove this?

Reply:
Using a local var would mean having to pass it between `setUpMocksAndFakes` and `testSend`, which is used by most of the tests.